src/Queries/Traits/HasRelatedWith.php

```diff
@@ -12,7 +12,6 @@
 trait HasRelatedWith
 {
     /**
-     * @param Row|RowCollection|Table
      * @param mixed $related
```

Comment:
The right signature is the first one (`Row|RowCollection|Table`)

Reply:
Then the `if ($related instanceof Table) {` line below is in vain as it'll be always true.